__init__.py

```diff
@@ -19,8 +19,6 @@ class LaughSkill(OVOSSkill):
     def sounds_dir(self) -> str:
         """Path to the sounds directory."""
         default = join(dirname(__file__), "sounds")
-        if not self.settings.get("sounds_dir"):
-            self.sounds_dir = default
```

Comment:
this should not be saved to file, was causing me weird bugs because it was pointing to a non existing venv from a previous install

Reply:
Not sure what you mean - this is to tell the skill where to find the sounds, defaulting to the location of the skill's init file + `sounds`? Why would it be pointing someplace else unless the config was explicitly pointing to the wrong location?

Reply:
For anyone checking this out in the future, this section was persisting the specific location of the `__init__.py` file in settings.json. By removing it this allows the property to remain dynamic if moving to a new virtual environment